### PR TITLE
Package build

### PR DESCRIPTION
**Pull request by mbivolan**

### CODE REVIEW

File: scripts/package_building/build_artifacts.sh [hunk 1]
```diff
@@ -68,13 +76,20 @@ function get_sources_http (){
     #
     base_dir="$1"
```
Comment (papagalu) [inline scripts/package_building/build_artifacts.sh]: you can use local

File: scripts/package_building/build_artifacts.sh [hunk 2]
```diff
@@ -111,10 +126,17 @@ function get_sources_local (){
     #
```
Comment (papagalu) [inline scripts/package_building/build_artifacts.sh]: ditto

File: scripts/package_building/deps-lis/rpm/lis-daemon.spec [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    echo "Stopping FCOPY Daemon...."
+    systemctl stop hv_fcopy_daemon
+    echo "Stopping VSS Daemon...."
+    systemctl stop hv_vss_daemon
+    rm -rf %{_sharedstatedir}/hyperv || :
+fi
+
+%post
+if [ $1 > 1 ] ; then
+        # Upgrade
```
Comment (papagalu) [inline scripts/package_building/deps-lis/rpm/lis-daemon.spec]: indentation

File: scripts/package_building/utils.sh [hunk 4]
```diff
@@ -158,3 +158,22 @@ pushd() {
...
+        branch="$git_branch"
+    fi
+    echo $branch
+}
+
+copy_artifacts() {
+    artifacts_folder=$1
+    destination_path=$2
+    
+    cp "$artifacts_folder"/*.rpm "$destination_path" 2> /dev/null || true
```
Comment (papagalu) [inline scripts/package_building/utils.sh]: why do you do || true?
what if the copy failed?

Comment (mbivolan) [reply scripts/package_building/utils.sh]: one of them is supposed to fail , and both cps will fail only if something goes wrong somewhere else in script

Comment (papagalu) [reply scripts/package_building/utils.sh]: okay, but don't make everything pass
i would suggest removing || true

and where you use this function and expect a fail, treat it over there.

Comment (ader1990) [reply scripts/package_building/utils.sh]: I suggest making a test before, if the files exist and then copy them over

**Pull request closed without merging**